tests/unit/core/ov/ops/test_ov_ops_infrastructures.py

```diff
@@ -107,5 +107,6 @@ def test_from_ov(self):
         op = ConstantV0.from_ov(op_ov)
         assert isinstance(op, ConstantV0)
         assert op.attrs.shape == self.shape
-        assert op.attrs.is_parameter
-        assert isinstance(op.data, torch.nn.parameter.Parameter)
+        # This testcase is not valid anymore, because OV replaces it with a constant
+        # assert op.attrs.is_parameter
```

Comment:
If it isn't necessary, how about removing it?

Reply:
Done